examples/templates/docker/main.tf

```diff
@@ -187,6 +187,12 @@ resource "docker_container" "workspace" {
     volume_name    = docker_volume.home_volume.name
     read_only      = false
   }
+  # Uncomment this to allow higher network speeds. Be aware this has security implications.
+  # See: https://man7.org/linux/man-pages/man7/capabilities.7.html
+  # capabilities {
+  #   add = ["CAP_NET_ADMIN"]
+  # }
```

Comment:
Still missing the env var right?

Reply:
Honestly I'll just remove for now until I un-featureflag it